dashboard/client/src/components/ProgressBar.component.test.tsx

```diff
@@ -0,0 +1,43 @@
+import { render, screen } from "@testing-library/react";
+import React from "react";
+import { ProgressBar } from "./ProgressBar";
+
+describe("ProgressBar", () => {
+  it("renders", async () => {
+    render(
+      <ProgressBar
+        progress={[
+          {
+            color: "blue",
+            label: "in progress",
+            value: 2,
+          },
+          {
+            color: "red",
+            label: "error",
+            value: 5,
+          },
+          {
+            color: "green",
+            label: "success",
+            value: 3,
```

Comment:
For tasks from jobs that are finished, how about we show them as "CANCELLED"?  That would be a workaround for any incorrect metrics from dead jobs we haven't cleaned up.

So basically, if the job is finished, the task states can only be in {FINISHED, CANCELLED}.